Lib/test/test_capi/test_list.py

```diff
@@ -342,3 +342,11 @@ def test_list_extend(self):
 
         # CRASHES list_extend(NULL, [])
         # CRASHES list_extend([], NULL)
+
+    def test_list_fromarraymoveref(self):
+        # Test PyList_FromArrayMoveRef()
+        list_fromarraymoveref = _testcapi.list_fromarraymoveref
+
+        lst = [object() for _ in range(5)]
```

Comment:
I would use the following example:
```suggestion
        lst = [[i] for _ in range(5)]
```
Better repr.

Reply:
First, I used `list("abdef")`, but all items are immortal and so tests didn't detect refcount bugs in my C code. I used `object()` to make sure that each object is unique.

I don't think that repr is important here, the test is not supposed to fail :-)